app/sidekiq/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -193,6 +201,36 @@ def handle_errors(submission, error)
         non_retryable_error_handler(submission, e)
       end
 
+      def handle_lighthouse_errors(submission, error) # rubocop:disable Metrics/MethodLength
+        if error.instance_of?(Common::Exceptions::UnprocessableEntity)
+          error_clone = error.deep_dup
+          upstream_error = error_clone.errors.first.stringify_keys
+          unless (upstream_error['source'].present? && upstream_error['source']['pointer'].present?) ||
+                 upstream_error['detail'].downcase.include?('retries will fail')
+            error = Common::Exceptions::UpstreamUnprocessableEntity.new(errors: error.errors)
+          end
+        end
+        raise error
+      rescue Common::Exceptions::BackendServiceException,
+             Common::Exceptions::Unauthorized, # 401 (UnauthorizedError?)
+             # 422 (UpstreamUnprocessableEntity, i.e. EVSS container validation)
+             Common::Exceptions::UpstreamUnprocessableEntity,
+             Common::Exceptions::TooManyRequests, # 429
+             Common::Exceptions::ClientDisconnected, # 499
+             Common::Exceptions::ExternalServerInternalServerError, # 500
+             Common::Exceptions::NotImplemented, # 501
+             Common::Exceptions::BadGateway, # 502
+             Common::Exceptions::ServiceUnavailable, # 503 (ServiceUnavailableException?)
+             Common::Exceptions::GatewayTimeout, # 504 (already here)
```

Comment:
Look like some stray comments? Could be worth for prosperity to finish these thoughts, but otherwise not super consequential